activerecord/lib/active_record/relation/finder_methods.rb

```diff
@@ -166,7 +166,7 @@ def last(limit = nil)
         if order_values.empty? && primary_key
           order(arel_table[primary_key].desc).limit(limit).reverse
         else
-          to_a.last(limit)
+          reverse_order.limit(limit).to_a.reverse
```

Comment:
Nvm, I understand need to get the last 3 from sql but order them back in original order